Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -1249,7 +1249,7 @@ def test_http_requests_with_retry_sanity(self, method):
     ]
 
     @pytest.mark.parametrize('method, status', RETRIES_NEGATIVE_TESTS_INPUT)
-    def test_http_requests_with_retry_negative_sanity(self, method, status):
+    def test_http_requests_with_retry_negative_sanity(self, method, status, requests_mock):
```

Comment:
Using requests_mock replaces the HTTPAdapter with a different one and basically make the test redundant since it is not testing the retry mechanism

Reply:
You can still reduce the backoff time of course.

Reply:
since the backoff is exponential, You can reduce the retries from 3 to 1, that will also reduce most of the time